README.md

```diff
@@ -23,31 +23,33 @@ let entryTypes = [Author.self, Category.self, Post.self]
 let store = CoreDataStore(context: self.managedObjectContext)
 let persistenceModel = PersistenceModel(spaceType: SyncInfo.self, assetType: Asset.self, entryTypes: entryTypes)
 
+// Initialize the Contentful.Client with a persistenceIntegration which will receive messages about changes when calling `sync methods`
+self.client = Client(spaceId: "<YOUR_SPACE_ID>", accessToken: "<YOUR_ACCESS_TOKEN>")
+
 // Create the manager.
-let syncManager = SynchronizationManager(persistenceStore: self.store, persistenceModel: persistenceModel)
+self.syncManager = SynchronizationManager(client: self.client,
+                                          localizationScheme: LocalizationScheme.all, // Save data for all locales your Space supports.
```

Comment:
For later: We should start aligning on how we spell things. I think marketing prefers to only capitalise new concepts, and a space should not be a new concept. But since we are not in alignment yet, I just wanted to raise awareness ... :)